src/main/resources/META-INF/keycloak.json

```diff
@@ -0,0 +1,8 @@
+{
+	"realm": "customer",
+	"auth-server-url": "https://sso.dev.cloud.retest.org/auth",
```

Comment:
There it says`dev`, in https://github.com/retest/recheck/pull/278/files#diff-0c94201b144c8886a305248889d51c84R237 not?